autoware_launch/CMakeLists.txt

```diff
@@ -7,6 +7,16 @@ ament_auto_find_build_dependencies()
 if(BUILD_TESTING)
   find_package(ament_lint_auto REQUIRED)
   ament_lint_auto_find_test_dependencies()
+
+  # static centerline optimizer
+  add_launch_test(
+    test/static_centerline_optimizer/test_static_centerline_optimizer.test.py
```

Comment:
[Question] Why is it impossible to test this in the `static_centerline_optimizer` package?

Reply:
Sorry to be late.

Do you mean this one?
In the static_centerline_optimizer, the node itself is tested.
https://github.com/autowarefoundation/autoware.universe/blob/main/planning/static_centerline_optimizer/test/test_static_centerline_optimizer.test.py

FYI: In autoware_launch, we should test not the node but launch.xml to launch the node which designates yaml file. That's why I make this PR draft.

Reply:
@takayuki5168 Thank you for the information, but I'm sorry it's still unclear to me.
- What is `launch.xml to launch the node which designates yaml file` in this case? Could you make it a bit more specific?
- What's the difference between https://github.com/autowarefoundation/autoware.universe/blob/main/planning/static_centerline_optimizer/test/test_static_centerline_optimizer.test.py and [autoware_launch/test/static_centerline_optimizer/test_static_centerline_optimizer.test.py](https://github.com/autowarefoundation/autoware_launch/pull/290/files#diff-17f171f631da095225016d0d148f894927f705c83b8cfe444f551c9f708792ac)? They look to be almost the same.

Reply:
@kenji-miyake 

> What is launch.xml to launch the node which designates yaml file in this case? 

This one.
https://github.com/autowarefoundation/autoware.universe/blob/main/planning/static_centerline_optimizer/launch/static_centerline_optimizer.launch.xml

> What's the difference

Currently, the difference is only whether launch config or node config is used as a yaml file.
Anyway, the current PR is under edit, so it would be better to discuss again after I fix. Sorry about that.

Reply:
> This one.
https://github.com/autowarefoundation/autoware.universe/blob/main/planning/static_centerline_optimizer/launch/static_centerline_optimizer.launch.xml

Are you trying to test this launch file? If so, where is it called in this pull request?

> Currently, the difference is only whether launch config or node config is used as a yaml file from the test.
Anyway, the current PR is under edit, so it would be better to discuss again after I fix. Sorry about that.

Okay. I think it's good to ask for a review after it's somewhat ready. If it's still a very draft, we should discuss it on an issue.